themes/bootstrap3/templates/Recommend/SideFacets.phtml

```diff
@@ -123,6 +123,17 @@ $(document).ready(function() {
   .on('slide', fillTexts)
   .data('slider');
 });
+
+$('#{$this->escapeHtmlAttr($title)}from, #{$this->escapeHtmlAttr($title)}to').change(function () {
+  $('#{$this->escapeHtmlAttr($title)}dateSlider').slider(
+    'setValue',
+    [
+      isNaN(Number($('#{$this->escapeHtmlAttr($title)}from').val())) || Number($('#{$this->escapeHtmlAttr($title)}from').val()) <= 0 ? {$min} : Number($('#{$this->escapeHtmlAttr($title)}from').val()),
+      isNaN(Number($('#{$this->escapeHtmlAttr($title)}to').val())) || Number($('#{$this->escapeHtmlAttr($title)}to').val()) <= 0 ? {$max} : Number($('#{$this->escapeHtmlAttr($title)}to').val())
```

Comment:
I'd recommend breaking this into smaller steps for readability. Also, saving `Number($('#{$this->escapeHtmlAttr($title)}from').val())` to a variable will prevent from repeating a jQuery search and conversion twice.